man/systemd.network.xml

```diff
@@ -208,6 +208,20 @@
           integer in the range 0—4294967294. Defaults to unset.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>TransmitQueues=</varname></term>
+        <listitem>
+          <para>Specifies the devices's number of transmit queues. An unsigned
```

Comment:
(doesn't matter, but the "unsigned" can be removed here, it's implied by the `1-4096` range)